### PR TITLE
Add remappings.txt

### PR DESCRIPTION
**Pull request by maurelian**
**Description**

Just writes the output of `forge remappings` to `remappings.txt` to facilitate easier usage of pyrometer.

### CODE REVIEW

**Review by tynes — COMMENTED** [review]
The annoying thing is that this could get out of date

File: packages/contracts-bedrock/remappings.txt [hunk 1]
```diff
@@ -0,0 +1,9 @@
+@cwia/=node_modules/clones-with-immutable-args/src/
+@openzeppelin/=node_modules/@openzeppelin/
```
Comment (tynes) [inline packages/contracts-bedrock/remappings.txt:2]: Is line 2 a dead remapping?

**Comment by tynes** [comment]
Maybe a npm script for generating this would be useful? Do you think its also too much to fail ci if there is a diff?

**Comment by maurelian** [comment]
> Maybe a npm script for generating this would be useful? Do you think its also too much to fail ci if there is a diff?

I'm open to that. My thought was that I wasn't sure about adding that kind of scaffolding around this as there's no firm commitment to using pyrometer, though there does seem to be enthusiasm. 

I think we should either
1. not do this
2. add the npm script and CI check. 

LMK what you think.

**Comment by refcell** [comment]
I think we should try it out and if we run into pain points with this wrt ci, we can remove.

**Pull request closed without merging**